doc/developer-guide/api/functions/TSUrlHostGet.en.rst

```diff
@@ -72,6 +76,8 @@ scheme.
 :arg:`offset` within the marshal buffer :arg:`bufp`. If there is no explicit
 port number in the URL, zero is returned.
 
+
+
```

Comment:
What do these two additional blank lines do?

Reply:
I must have done it by mistake. I will remove these lines